README.md

```diff
@@ -1,4 +1,75 @@
-# Easy Connect - Easily add all supported connectivity methods to your mbed OS project
+# Easy Connect - Easily add all supported connectivity methods to your Mbed OS project
+
+## Deprecation note
+
+Please note that starting with [Mbed OS 5.10](https://github.com/ARMmbed/mbed-os/tree/mbed-os-5.10) onwards the network driver can be instantiated using `get_default_instance()`. 
+
```

Comment:
... and onwards, you can instantiate the network driver using ...

Reply:
Why are we writing this document on how to use the new API, it is already done here. Simply re-direct users to the correct documentation link. 
https://os-doc-builder.test.mbed.com/docs/development/apis/network-interfaces.html
https://os-doc-builder.test.mbed.com/docs/development/reference/configuration-connectivity.html#selecting-the-default-network-interface
It will save duplication of same thing.

Reply:
I can't link to any of that. At least, not yet.